src/main/java/com/WWI16AMA/backend_api/PlaneLog/PlaneLogController.java

```diff
@@ -0,0 +1,45 @@
+package com.WWI16AMA.backend_api.PlaneLog;
+
+import com.WWI16AMA.backend_api.Member.MemberRepository;
+import com.WWI16AMA.backend_api.Plane.Plane;
+import com.WWI16AMA.backend_api.Plane.PlaneRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+
+
+@RestController
+@RequestMapping(path = "planeLog")
+public class PlaneLogController {
+
+    @Autowired
+    private PlaneRepository planeRepository;
+    private MemberRepository memberRepository;
+
+
+    @GetMapping(path = "/{id}")
+    public ResponseEntity<List> info(@PathVariable int id) {
+
+        return new ResponseEntity<>(planeRepository.findById(id)
+                .orElseThrow(() -> new NoSuchElementException("Plane with the id " + id + " does not exist")).getPlaneLog().getEntries(), HttpStatus.OK);
+    }
+
+    @PostMapping(path = "/{id}")
+    public List addPlaneLogEntry(@RequestBody PlaneLogEntry entry, @PathVariable int id) {
+        Plane plane = planeRepository.findById(id)
+                .orElseThrow(() -> new NoSuchElementException("Plane with the id " + id + " does not exist"));
+
+        PlaneLog planeLog = plane.getPlaneLog();
+        planeLog.addPlaneLogEntry(entry);
```

Comment:
Vielleicht hier ein paar Überprüfungen, zb `startCount < endCount` und, wenn tryharded: `refuelDateTime` muss zeitlich nach dem letzten tanken und vor "jetzt" liegen..
Bei Fehlern vllt eine `ConstraintViolationException` werfen, die wird dann in `AppControllerAdvice abgefangen und mit ner 400 zurückgeschickt..